nautobot_bgp_models/filters.py

```diff
@@ -22,21 +29,19 @@ class AutonomousSystemFilterSet(
         label="Search",
     )
 
-    tag = TagFilter()
-
-    def search(self, queryset, name, value):  # pylint: disable=unused-argument,no-self-use
+    def search(self, queryset, name, value):  # pylint: disable=unused-argument
         """Free-text search method implementation."""
         if not value.strip():
             return queryset
         return queryset.filter(Q(asn__icontains=value) | Q(description__icontains=value)).distinct()
 
     class Meta:
         model = models.AutonomousSystem
-        fields = ["id", "asn", "status"]
+        fields = ["id", "asn", "status", "tags"]
```

Comment:
```suggestion
        fields = models.AutonomousSystem().all_fields
```

@glennmatthews thoughts on moving to this pattern?